### PR TITLE
Fix features build

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin

### CODE REVIEW

**Comment by EPashkin** [comment]
Travis failing here

**Comment by GuillaumeGomez** [comment]
Arf, forgot to add a file!

File: src/mark_attributes.rs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+use glib::translate::*;
+use glib::object::IsA;
+
+use gdk_pixbuf;
+use gtk;
+
+use MarkAttributes;
+
+// We don't have the "real" ffi type so let's just cast like that...
+unsafe fn const_to_mut<T>(t: *const T) -> *mut T {
```
Comment (EPashkin) [inline src/mark_attributes.rs]: It is same as glib::mut_override ?

**Comment by EPashkin** [comment]
Also IMHO better add .gitignore

File: src/mark_attributes.rs (hunk 1 above)
Comment (GuillaumeGomez) [inline src/mark_attributes.rs]: Ah maybe! I'll check this evening.

**Comment by GuillaumeGomez** [comment]
> Also IMHO better add .gitignore

Indeed. I'll add it as well.

**Comment by GuillaumeGomez** [comment]
Added the `.gitignore` and used `mut_override`.

**Comment by EPashkin** [comment]
Maybe it also need some `min_cfg_version` in Gir.toml?
Other looks good.

**Comment by EPashkin** [comment]
@GuillaumeGomez You sure that sourceview versions and gtk's completely independent?

**Comment by GuillaumeGomez** [comment]
Not at all. I think sourceview is completely dependent. I'm adding the minimum version.

**Comment by EPashkin** [comment]
Submodules still points to wrong commits

**Comment by GuillaumeGomez** [comment]
Updated submodules and regenerated with 3.0 as minimum version.

**Comment by EPashkin** [comment]
Gir now pointed to d121f7e0, generated with fdeaa47

**Comment by GuillaumeGomez** [comment]
Fixed.

**Comment by EPashkin** [comment]
Will fails due bitflags!="0.9"

**Comment by GuillaumeGomez** [comment]
Seems CI is ok. :)

**Comment by EPashkin** [comment]
👍

**Pull request merged**